metadata-ingestion/docs/sources/sac/sac_pre.md

```diff
@@ -0,0 +1,45 @@
+## Configuration Notes
+
+1. Refer to [Manage OAuth Clients](https://help.sap.com/docs/SAP_ANALYTICS_CLOUD/00f68c2e08b941f081002fd3691d86a7/4f43b54398fc4acaa5efa32badfe3df6.html) to create an OAuth client in SAP Analytics Cloud. The OAuth client is required to have the following properties:
+
+    - Purpose: API Access
+    - Access:
+        - Data Import Service
+    - Authorization Grant: Client Credentials
+
+2. Maintain connection mappings (optional):
+
+To map individual connections in SAP Analytics Cloud to platforms, platform instances or environments the `connection_mapping` configuration can be used within the recipe:
+
+```yaml
+connection_mapping:
+    MY_BW_CONNECTION:
+        platform: bw
+        platform_instance: PROD_BW
+        env: PROD
+    MY_HANA_CONNECTION:
+        platform: hana
+        platform_instance: PROD_HANA
+        env: PROD
+```
+
+The key in the connection mapping is the technical name of the connection resp. its id.
```

Comment:
bit confused by this line

Reply:
I have changed it a bit, not sure if it is less confusing then...

You have to maintain the connections to the different systems in the SAC...there you have to specify a name: https://developers.sap.com/tutorials/hana-cloud-analytics-cloud-consume-2/jcr:content.github-proxy.1633841405.file/step7-I7.png

The key in the connection mapping dictionary is this name and is used to find the connection in the dictionary...